_/scss/modules/_search.scss

```diff
@@ -0,0 +1,37 @@
+//----------------------------------------------------------------
+//  Search
+//----------------------------------------------------------------
+.search {
+    position: relative;
+    &:after {
+        position: absolute;
+        top: 11px;
+        right: 12px;
+
+        display: block;
+
+        width: 20px;
+        height: 20px;
+
+        content: '';
+        pointer-events: none;
+
+        opacity: .6;
+        background: url(_/images/search.svg) center no-repeat;
+        background-size: contain;
+    }
+}
+
+.search--input {
+    font-size: 16px;
+    line-height: 1.5;
+
+    width: 100%;
+    padding: 8px 36px 8px 12px;
+
+    border: 1px solid #ddd;
+    &:focus {
```

Comment:
Not crucial, more of a style thing - could we maybe put a line break before the pseudo element selectors? (line 6 and 33)